contracts/protocol/contracts/ERC1155_Starter/ScoutGameStarterNFTProxy.sol

```diff
@@ -31,27 +23,24 @@ contract ScoutGameStarterPackNFTUpgradeable {
             paymentTokenAddress != address(0),
             "Invalid payment token address"
         );
-        MemoryUtils.setAddress(MemoryUtils.ADMIN_SLOT, msg.sender);
-        MemoryUtils.setAddress(
+        MemoryUtils._setAddress(MemoryUtils.ADMIN_SLOT, _msgSender());
+        MemoryUtils._setAddress(
             MemoryUtils.IMPLEMENTATION_SLOT,
             implementationAddress
         );
-        MemoryUtils.setAddress(
-            MemoryUtils.PAYMENT_ERC20_TOKEN_SLOT,
+        MemoryUtils._setAddress(
+            MemoryUtils.CLAIMS_TOKEN_SLOT,
             paymentTokenAddress
         );
-        MemoryUtils.setAddress(
+        MemoryUtils._setAddress(
             MemoryUtils.PROCEEDS_RECEIVER_SLOT,
             _proceedsReceiver
         );
 
-        MemoryUtils.setUint256(MemoryUtils.PRICE_INCREMENT_SLOT, 2000000);
-
-        MemoryUtils.setString(MemoryUtils.TOKEN_NAME, _tokenName);
-        MemoryUtils.setString(MemoryUtils.TOKEN_SYMBOL, _tokenSymbol);
+        MemoryUtils._setUint256(MemoryUtils.PRICE_INCREMENT_SLOT, 2000000);
```

Comment:
this is still using the 6 decimal places. if we plan to keep the starter nft increments static we should use the 18 decimal one